MachineLearning_Engine/Compute/InstallPythonML.cs

```diff
@@ -33,7 +33,7 @@ public static partial class Compute
         [Description("MachineLearning_Toolkit\nMethod used to create the Python environment used to run all Python scripts within this toolkit.")]
         [Input("run", "Starts the installation of the toolkit if true. Stays idle otherwise.")]
         [Output("pythonEnvironment", "The MachineLearning_Toolkit Python environment.")]
-        public static PythonEnvironment MachineLearningToolkitPythonEnvironment(bool run = false)
+        public static PythonEnvironment InstallPythonML(bool run = false)
```

Comment:
Didn't want to include the word `Env` in it? Thought it was better to have it - we're not simply installing a Python distribution, we are actually creating an environment for it no?

Reply:
Just wondering as the other day we said we'd go for `InstallPyEnv_ML`, `InstallPyEnv_LBT`, etc.
I actually like the fact you included the entire word "Python" in the name (easier to search), but I think we should mention "Environment" or "Env" for clarity?